include/zephyr/logging/log_backend_ble.h

```diff
@@ -30,7 +30,7 @@ typedef void (*logger_backend_ble_hook)(bool backend_status, void *ctx);
 /**
  * @brief Allows application to add a hook for the status of the BLE
  *        logger backend.
- * @details The BLE logger backend is enabled or disabled auomatically by
+ * @details The BLE logger backend is enabled or disabled automatically by
```

Comment:
Please fix: Should probably omit this change, since it's unrelated.

Reply:
Will do!